doc/reference/configuration/configuration_reference.rst

```diff
@@ -1249,6 +1217,149 @@ The ``database`` section defines database-specific configuration parameters, suc
     | Default: false
     | Environment variable: TT_DATABASE_USE_MVCC_ENGINE
 
+..  _configuration_reference_fiber:
+
+fiber
+-----
+
+The ``fiber`` section describes options related to configuring :ref:`fibers, yields, and cooperative multitasking <concepts-coop_multitasking>`.
+
+..  NOTE::
+
+    ``fiber`` can be defined in any :ref:`scope <configuration_scopes>`.
+
+* :ref:`fiber.io_collect_interval <configuration_reference_fiber_io_collect_interval>`
+* :ref:`fiber.too_long_threshold <configuration_reference_fiber_too_long_threshold>`
+* :ref:`fiber.worker_pool_threads <configuration_reference_fiber_worker_pool_threads>`
+* :ref:`fiber.slice.* <configuration_reference_fiber_slice_options>`
+* :ref:`fiber.top.* <configuration_reference_fiber_top_options>`
+
+..  _configuration_reference_fiber_io_collect_interval:
+
+..  confval:: fiber.io_collect_interval
+
+    The :ref:`fiber <app-fibers>` will sleep for the defined number of seconds
+    between iterations of the event loop. Can be used to reduce CPU load in deployments
+    where the number of client connections is large, but requests are not so frequent
+    (for example, each connection issues just a handful of requests per second).
+
+    |
+    | Type: number
+    | Default: box.NULL
+    | Environment variable: TT_FIBER_IO_COLLECT_INTERVAL
+
+..  _configuration_reference_fiber_too_long_threshold:
+
+..  confval:: fiber.too_long_threshold
+
+    If processing a request takes longer than the given value (in seconds),
+    the fiber warns about it in the log.
+    Has effect only if :ref:`log.level <configuration_reference_log_level>`
+    is greater than or equal to 4 (``warn``).
+
+    |
+    | Type: number
+    | Default: 0.5
+    | Environment variable: TT_FIBER_TOO_LONG_THRESHOLD
+
+..  _configuration_reference_fiber_worker_pool_threads:
+
+..  confval:: fiber.worker_pool_threads
+
+    The maximum number of threads to use during execution
+    of certain internal processes (currently
```

Comment:
I'd use `for example` to protect us from possible changes here. Developers can apply this limit to other processes and say nothing to us.

```suggestion
    of certain internal processes (for example,
```